version.properties

```
@@ -1,9 +1,4 @@
 #Mon, 30 Aug 2021 11:48:28 +0200
 
-# Version Information for Vanilla / Release builds
 versionName=20.5-rc-1
 versionCode=1259
```

Following the good point made by @oguzkocer  in his review comment, I think just manually updating that line would be enough to avoid developers risking to encounter a `versionCode` decrease (since  `1260` was already previously used by `wasabi` flavor)

```suggestion
versionCode=1260
```

---

PS: I plan to merge `release/20.5` into `trunk` once this PR lands, so that the last 3 tooling-only PRs that happened in `release/20.5` recently would land in `trunk` before I even do the next beta (which would also help isolating those tooling PR landings from the future post-beta PR that will happen next later in the sprint when a new beta will be needed)

---

This works for me.

I don't like having a disconnection between the `versionName` and `versionCode`, so I was a bit hesitant to suggest this standalone bump. However, I admit that it's the most practical way to address the issue. Maybe you can leave a note in the commit message about why there is a separate bump for `versionCode` and that the artifact for it doesn't exist in Play Store, just as a breadcrumb in case we land in this commit while investigating the git history?

> no need to actually do a new beta build

Yeap, no need for a new beta, this bump is just to simplify the developer experience.

> and again, good thoughts and thanks for thinking about that case! 🙌

I am glad to hear that you found it useful 🙂

---

> I don't like having a disconnection between the versionName and versionCode, so I was a bit hesitant to suggest this standalone bump.

Yeah, I agree that  I'm not a fan either of having the `versionName` and `versionCode` in `version.properties` not match to the actual values we used for the latest upload of a build to the PlayStore.

> However, I admit that it's the most practical way to address the issue. 

Yeah I think this is acceptable to make addressing this easier indeed. We can live we that especially since it will re-sync next time we do a beta — which will be Wednesday next week… at the latest)

> Maybe you can leave a note in the commit message about why there is a separate bump for versionCode and that the artifact for it doesn't exist in Play Store, just as a breadcrumb in case we land in this commit while investigating the git history?

Totally 👍 . Did that in [020439b](https://github.com/wordpress-mobile/WordPress-Android/pull/17023/commits/020439bdd505c10200ffecbbd2c029d4dea1cdfe); hopefully I explained the whole thing well enough in that commit message.